provider/docker/config_container.go

```diff
@@ -214,6 +214,23 @@ func getHealthCheck(container dockerData) *types.HealthCheck {
 	}
 }
 
+func getBuffering(container dockerData) *types.Buffering {
+	enabled := label.GetBoolValue(container.Labels, label.TraefikBackendBufferingEnabled, false)
```

Comment:
Could you remove the `label.TraefikBackendBufferingEnabled` and instead use `label.HasPrefix(...)`

Same for all providers.